doc/Building_FRR_on_Ubuntu1604.md

```diff
@@ -0,0 +1,111 @@
+Building FRR on Ubuntu 12.04LTS from Git Source
+===============================================
+
+- MPLS is not supported on `Ubuntu 12.04` with default kernel. MPLS requires 
```

Comment:
See https://lists.ubuntu.com/archives/ubuntu-announce/2016-November/000215.html
They talk about an optional 4.8 kernel, but not default. The URL I mention allows to download newer Ubuntu Kernels as well (and they work fine - I'm using them for my testing)